src/Tribe/Cost_Utils.php

```diff
@@ -346,7 +346,7 @@ public function parse_cost_range( $costs, $max_decimals = null, $sort = true ) {
 		}
 
 		$output_costs = [];
-		$costs        = call_user_func_array( 'array_merge', $costs );
+		$costs        = call_user_func_array( 'array_merge', [ $costs ] );
```

Comment:
If `$costs` is already an array here, this would cause a problem. I'd suggest the following:

```suggestion
		if ( ! empty( $costs ) && ! is_array( $costs ) ) {
			$costs        = call_user_func_array( 'array_merge', [ $costs ] );
		}
```

Reply:
I dug a bit more - the real issue turns out to be strings for keys.
`array_values` fixes that - and we don't use the keys anyway?

Paraphrased:

As a clarification for why this is failing, it's due to the new named argument assignment feature in PHP 8. In prior versions of PHP the keys were ignored and each item in your array passed to call_user_func_array was assigned in-order to the arguments of the function you were attempting to call. Basically: array_merge($array['one'], $array['two'])

As of PHP 8.0, it's trying to match the keys of the array to the function argument name, you'll note that if you change one of the indexes to "array" and remove the other, this works on the latest PHP 8 RC, as the first argument in array_merge is labelled as '$array' and thus the array index of your array gets mapped to that. Basically: array_merge(one: $array['one'], two: $array['two'])